website/docs/reference/runtimes/index.mdx

```diff
@@ -15,7 +15,7 @@ The status represents the maturity of the runtime implementation:
 Submit your request and vote for your preferred ones [here](https://github.com/metatypedev/metatype/discussions/305).
 :::
 
-| Runtime                         | Description                                                | Runtime version | Status | Typegate version |
+|             Runtime             |                        Description                         | Runtime version | Status | Typegate version |
 | ------------------------------- | ---------------------------------------------------------- | --------------- | ------ | ---------------- |
 | [Prisma](./runtimes/prisma)     | Query SQL and no-SQL databases with Prisma                 | 4.17.0          | beta   | >0.1.0           |
 | [Deno](./runtimes/deno)         | Execute Typescript functions with NPM or deno dependencies | 1.35.0          | beta   | >0.1.0           |
```

Comment:
I think we might need to add these lines in `dev/lock.yml`, to match the actual implemented version...

Reply:
I think it would be more work to try to match/replace these since these are only set once.